styles.css

```diff
@@ -424,3 +431,303 @@ footer hr {
   margin-top: 1vh;
   padding-top: 0%;
 }
+
+@media (min-width: 768px) {
```

Comment:
- [ ] Kindly review all of your margins, paddings, font-sizes, heights and widths, and every other attribute that you used `vh` or `vw` for sizing and change for the values that the Figma template provides (px in most cases). This is causing distortion on some sizes of screens and making a bad user experience when scrolling through your app.
![image](https://user-images.githubusercontent.com/61919617/171906840-6a889a5b-4887-4a3e-9715-176cbe382ab1.png)
![image](https://user-images.githubusercontent.com/61919617/171906934-408aed0c-7ccd-4b66-9d54-084144f7a93a.png)
![image](https://user-images.githubusercontent.com/61919617/171906978-02a78d3f-66f8-458c-b4ed-954bb2853cfa.png)
![image](https://user-images.githubusercontent.com/61919617/171907052-4227b6be-e96f-4a7b-a44c-d5882e5b120e.png)
![image](https://user-images.githubusercontent.com/61919617/171907112-743de517-bf09-40fd-9666-f5434cda0b75.png)